modules/admin-ui-picard/app/src/thunks/seriesDetailsThunks.js

```diff
@@ -0,0 +1,250 @@
+
+        let data = await axios.get(`admin-ng/series/${id}/theme.json`);
+
+        const themeResponse = await data.data;
+
+        let seriesTheme = '';
+
+        // check if series has a theme
+        if (!_.isEmpty(themeResponse)) {
+            // transform response for further use
+            seriesTheme = (transformToIdValueArray(themeResponse))[0].value;
+        }
+
+        dispatch(loadSeriesDetailsThemeSuccess(seriesTheme));
+    } catch (e) {
+        dispatch(loadSeriesDetailsFailure());
+    }
+}
+
+// fetch names of possible themes from server
+export const fetchNamesOfPossibleThemes = () => async dispatch => {
+    try {
+        dispatch(loadSeriesDetailsThemeNamesInProgress());
+
+        let data = await axios.get('/admin-ng/resources/THEMES.NAME.json');
+
+        const response = await data.data;
+
+        // transform response for further use
+        let themeNames = transformToIdValueArray(response);
+
+        dispatch(loadSeriesDetailsThemeNamesSuccess(themeNames));
+    } catch (e) {
+        dispatch(loadSeriesDetailsThemeNamesFailure());
+    }
+}
+
+// update series with new metadata
+export const updateSeriesMetadata = (id, values) => async (dispatch, getState) => {
+    try {
+        let metadataInfos = getSeriesDetailsMetadata(getState());
+
+        let fields = [];
+        let updatedFields = [];
+
+        metadataInfos.fields.forEach(field => {
+            if (field.value !== values[field.id]) {
+                let updatedField = {
+                    ...field,
+                    value: values[field.id]
+                }
+                updatedFields.push(updatedField);
+                fields.push(updatedField);
+            } else {
+                fields.push({...field});
+            }
+        });
+
+        let data = new URLSearchParams();
+        data.append("metadata",JSON.stringify([{
+            flavor: metadataInfos.flavor,
+            title: metadataInfos.title,
+            fields: updatedFields
+        }]));
+
+        await axios.put(`/admin-ng/series/${id}/metadata`, data);
+
+        // updated metadata in series details redux store
+        let seriesMetadata = {
+            flavor: metadataInfos.flavor,
+            title: metadataInfos.title,
+            fields: fields
+        };
+        dispatch(loadSeriesDetailsMetadataSuccess(seriesMetadata));
+    } catch (e) {
+        console.log(e);
+    }
+}
+
+export const updateSeriesAccess = (id, policies) => async (dispatch) => {
+    let data = new URLSearchParams();
+
+    let acls = {
+        acl: {
+            ace: policies
+        }
+    }
+    data.append("acl", JSON.stringify(acls));
+    data.append("override", true);
+
+    return axios.post(`admin-ng/series/${id}/access`, data, {
+        headers: {
+            'Content-Type': 'application/x-www-form-urlencoded'
+        }
+    })
+        .then(res => {
+            console.log(res);
+            dispatch(addNotification('info', 'SAVED_ACL_RULES'));
```

Comment:
This Notification should be displayed inside the modal (on the top of the tab), not in the lower right corner